### PR TITLE
Added quiet, less quiet modes

### PR DESCRIPTION
**Pull request by kxalex**
* Added quiet and less verbose modes.

### CODE REVIEW

**Comment by kxalex** [comment]
Hey @erik. Is everything good with this PR? Do you want me to add/change anything?

File: holepunch/__init__.py [hunk 1]
```diff
@@ -294,11 +308,11 @@ def holepunch(args):
...
-    print('Changes to be made to: {group_name} [{group_id}]'
-          '\n{hr}\n{perms}\n{hr}'.format(
-              hr='='*60, group_name=group['GroupName'],
-              group_id=group['GroupId'],
-              perms=json.dumps(new_perms, indent=4)))
+    log('Changes to be made to: {group_name} [{group_id}]'
+        '\n{hr}\n{perms}\n{hr}'.format(
+        hr='='*60, group_name=group['GroupName'],
+        group_id=group['GroupId'],
+        perms=json.dumps(new_perms, indent=4)))
```
Comment (erik) [inline holepunch/__init__.py]: Here I would argue that we would want to print this if `args['--yes'] == False`, even when using the quiet mode, as it is part of the confirmation

File: holepunch/__init__.py [hunk 2]
```diff
@@ -21,7 +21,9 @@
   -t --tcp               Open TCP ports to ingress [default].
   -u --udp               Open UDP ports to ingress.
   -y --yes               Don't prompt before writing rules.
-'''
```
Comment (erik) [inline holepunch/__init__.py]: nit: can you please revert the unrelated change? i know `black` etc. would make this `"""`, but as is this would be inconsistent in the rest of the file

File: holepunch/version.py [hunk 3]
```diff
@@ -1 +1 @@
-__version__ = '0.9.0'
```
Comment (erik) [inline holepunch/version.py]: please revert this, i'll bump the version when i release

File: README.md [hunk 4]
```diff
@@ -62,3 +62,13 @@ holepunch foo_bar --cidr=192.168.0.0/16 22 80
...
 ```
+
+Less verbose or quiet mode:
+
+```
+# Verbose mode
+holepunch foo_bar -v2 --cidr=192.168.0.0/16 22 80
+
+# Quiet mode
+holepunch foo_bar -v0 --cidr=2001:882f::1/128 443
```
Comment (erik) [inline README.md]: IMO this is overkill for how simple of a tool `holepunch` is. Using `-v 0` to disable messages with `-v 2` being the default feels a little off to me.

The intention of the messages that get printed currently wasn't debugging, but rather to make sure the user is confident they're making the correct changes. If any of the messages seem to be not useful, maybe we can remove or reword them instead.

I do like the idea of having a quiet mode, but I think my preference here would be to have two options: 

- print progress messages (default, no flag)
- don't print (maybe `-s, --silent` as in `curl`)

i.e. 

```
# Verbose output (default)
holepunch foo_bar --cidr=192.168.0.0/16 22 80

# Quiet output
holepunch foo_bar --silent --cidr=2001:882f::1/128 443
```

If we really need it, maybe with an additional `--debug` option to produce more information (possibly even enabling logging inside boto etc), but I'm not sure that's necessary.

**Review by erik — COMMENTED** [review]
Hey @kxalex, sorry for the review delay. Been away and wanted to wait until I had a chance to read it through. Thanks again for splitting up the PR from before.

I like the idea of a quiet mode, and would be happy to include it, but I left some comments on the interface. Let me know what you think
